Resources/views/ContentType/view_content_type.html.twig

```diff
@@ -23,13 +24,66 @@
 
 {% block content %}
     <section class="ez-serverside-content">
-        <div class="ez-table-data is-flexible">
-            <div class="ez-table-data-container">
+        <div class="context-information">
+            <p class="left modified">
```

Comment:
can skip `left`, no such class here and left is anyway default alignment on ltr ;)